Calories.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Sep 23 00:45:03 2020
+
+@author: olumideakinola
+"""
+
+from clarifai.rest import ClarifaiApp, Image as ClImage
+
+import wolframalpha
+
+
+class Calories:
+ 
+    
+    def __init__(self,image):
```

Comment:
`image_url` rather than `image`. naming seem minor, but it's pretty important

Reply:
You sha like stress